src/components/Modal/ConfirmStakingParams/index.tsx

```diff
@@ -95,34 +95,18 @@ const ConfirmStakingParamsModal: FC<
             authorizer: account as string,
           }}
         />
-        <BodySm color="gray.500" mt="10">
-          Provider, Beneficiary, and Authorizer addresses are currently set to
-          your wallet address.
-        </BodySm>
-        <BodyMd textAlign="center" mt="4" mb="6">
-          {account}
-        </BodyMd>
-        <Flex direction="column">
-          <Button
-            variant="link"
-            color={useColorModeValue("brand.500", "white")}
-            onClick={onToggle}
-            mb={3}
-            alignSelf="flex-end"
-            rightIcon={isOpen ? <BsChevronUp /> : <BsChevronDown />}
-          >
-            Customize these addresses
-          </Button>
-        </Flex>
-        <Collapse in={isOpen} animateOpacity>
-          <AdvancedParamsForm
-            innerRef={formRef}
-            formId="advanced-staking-params-form"
-            initialAddress={account as string}
-            onSubmitForm={onSubmit}
-            checkIfProviderUsed={checkIfProviderUsed}
-          />
-        </Collapse>
+        <Alert my={6}>
+          <AlertIcon />
+          Take note! These addresses cannot be changed later.
+        </Alert>
```

Comment:
Based on Figma views the Alert should have status Info and it should be "small".

We should probalby add `sizes` property to the theme in `components` repo and add two sized - large and small (these are the only two we have in Threshold designs). Although' I've tried doing that in Alert theme:

```
sizes: {
    sm: {
      fontSize: "sm",
    },
    lg: {
      fontSize: "lg",
    },
  },
  ```

and it didn't work when i add `size=sm` to the Alert (but it worked when I added fontSize="sm" to the Alert).

We should also reduce the size of the Alert Icon for small size.

We can just add:

```suggestion
        <Alert my={6} status="info" fontSize="sm">
          <AlertIcon />
          Take note! These addresses cannot be changed later.
        </Alert>
```

for now if the there will be any problems with above solution.